Makefile

```diff
@@ -1,7 +1,30 @@
-test-yes:
-	./tests/yes.sh
+# Export GO111MODULE=on to enable project to be built from within GOPATH/src
+export GO111MODULE=on
+GO_PACKAGES=$(shell go list ./... | grep -v vendor)
+.PHONY: lint \
+        deps-update \
+        vet
+vet:
+	go vet ${GO_PACKAGES}
 
-test-bm-arm-deployment:
-	/bin/bash tests/gpu-operator-arm-bm/uninstall-gpu-operator.sh
-	/bin/bash tests/gpu-operator-arm-bm/install-gpu-operator.sh
-	/bin/bash tests/gpu-operator-arm-bm/areweok.sh
+lint:
+	@echo "Running go lint"
+	scripts/golangci-lint.sh
+
+deps-update:
+	go mod tidy && \
+	go mod vendor
+
+install-ginkgo:
+	scripts/install-ginkgo.sh
+
+build-container-image:
+	@echo "Building container image"
+	podman build -t nvidiagpu:latest -f Dockerfile
```

Comment:
Should we have a full name such as `quay.io/rh-ecosystem-edge:nvidiagpu:latest`?

Reply:
Yes, once we have access to push to this registry

Reply:
Yes, we can update once we create the repo